tasks/st0317a_clean_cert.rb

```diff
@@ -52,7 +56,7 @@ def clean_cert(agent, cmd)
   results[agent] = {}
 
   if agent == Puppet[:certname]
-    results[agent][:result] = 'Refusing to remove the Puppet Primary Server certificate'
+    results[agent][:result] = '***THIS TASK IS NOW DEPRECATED - Please see README*** - Refusing to remove the Puppet Primary Server certificate'
```

Comment:
Same as the others, we shouldn't add the deprecation to the results of each certname we try to clean.  Please remove it from here.